bolt-design-system/bolt
# Fix missing images in card example

## Jira

- http://vjira2:8080/browse/BDS-928

## Summary

Replace missing images in personalized card example.

## Details
Personalized card example referenced images that are no longer available on production. These images have been replaced with local images that will not be removed.

## How to test

- Go to:
  - [personalized card test (client-rendered)](https://fix-vr-fix-image-paths.boltdesignsystem.com/pattern-lab/patterns/02-components-card-__tests__-personalized-card--client-rendered-personalized-card-example--client-rendered-tests/02-components-card-__tests__-personalized-card--client-rendered-personalized-card-example--client-rendered-tests.html).
  - [personalized card test (server-rendered)](https://fix-vr-fix-image-paths.boltdesignsystem.com/pattern-lab/patterns/02-components-card-__tests__-personalized-card--server-rendered-personalized-card-example--server-rendered-tests/02-components-card-__tests__-personalized-card--server-rendered-personalized-card-example--server-rendered-tests.html).
- Verify that, after a delay, the card content is replaced and new images appear.
